generators/ci-cd/templates/.gitlab-ci.yml.ejs

```diff
@@ -80,6 +80,15 @@ gradle-test:
 
 gradle-integration-test:
     stage: test
+<%_ if (databaseType === 'sql') { _%>
```

Comment:
Maybe we can make the out of box experience with gitlab.com better (like we did just now for other dbs which already require testcontainers).
E.g.  check if we run inside docker, start the dind service and use the correct spring profile (so basically not just a comment) what do you think?

Reply:
I think this is already added as per https://github.com/jhipster/generator-jhipster/pull/11603. You probably just need to add the extra condition. :smile:

Reply:
This is already added as per #11603, you probably just need to add the extra condition. smile

Reply:
Since the work on the gitlab-ci.yml has already been done in #11603 I think I can fix it accordingly. 

@SudharakaP Only one thing. I see that DinD has been activated in the `gradle-test` job, but it should probably be moved in the `gradle-integration-test` job, right?

Reply:
@ecostanzi : You are right on that. I am doing some further testing for the Gradle side in Gitlab. I'll do a PR today with those changes. :smile:

Reply:
@ecostanzi : Done at https://github.com/jhipster/generator-jhipster/pull/11614. Thanks for pointing it out.